package.json

```diff
@@ -6,20 +6,23 @@
   "dependencies": {
     "bcrypt": "^1.0.2",
     "body-parser": "^1.17.1",
+    "cookie-parser": "^1.4.3",
     "dotenv": "^4.0.0",
     "express": "^4.15.2",
     "jsonwebtoken": "^7.3.0",
-    "mongoose": "^4.9.1"
+    "mongoose": "^4.9.1",
+    "morgan": "^1.8.1"
   },
   "devDependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
Using `mongoose`'s default promise library, yields the below error.
```
(node:10752) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead:
http://mongoosejs.com/docs/promises.html
```
`bluebird` was the suggested from the link.